modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -78,6 +78,7 @@ public BuildPluginDescriptorTask() {
 			Conf2ScopeMappingContainer.COMPILE);
 		_configurationScopeMappings.put(
 			"provided", Conf2ScopeMappingContainer.PROVIDED);
+		_pomRepositories.put("provided", Conf2ScopeMappingContainer.PROVIDED);
```

Comment:
Between the `_configurationScopeMappings` "block" and the `_pomRepositories` line, please leave an empty line: we're doing stuff of two different variables, so logically they're two different "blocks" of code.